internal/connect/product_test.go

```diff
@@ -73,6 +79,30 @@ func TestUnmarshalJSONscc(t *testing.T) {
 	}
 }
 
+func TestUnmarshalJSONBase(t *testing.T) {
+	jsn := `{"identifier": "product1", "base": true}`
+
+	var p Product
+	if err := json.Unmarshal([]byte(jsn), &p); err != nil {
+		t.Errorf("Error unmarshalling: %s", err)
+	}
+	if !p.IsBase {
+		t.Error("Expected p.IsBase == true, got false")
+	}
+}
+
+func TestUnmarshalJSONIsBase(t *testing.T) {
+	jsn := `{"identifier": "product1", "isbase": true}`
+
+	var p Product
+	if err := json.Unmarshal([]byte(jsn), &p); err != nil {
+		t.Errorf("Error unmarshalling: %s", err)
+	}
+	if !p.IsBase {
+		t.Error("Expected p.IsBase == true, got false")
+	}
+}
+
```

Comment:
should there be a test for `{"product_type": "base"}` too?

Reply:
added